src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpException.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Net.Http
+{
+    internal class WinHttpException : Win32Exception
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static int HRESULT_FROM_WIN32(int errorCode)
+        {
+            // Implementation of HRESULT_FROM_WINT32 macro
```

Comment:
@davidsh Nit: I believe the function name is enough to draw the conclusion from the comment.